src/Message/CancelSubscriptionsRequest.php

```diff
@@ -212,7 +238,7 @@ public function getData()
             'action' => $this->getFunction(),
             'force' => true,
             'disentitle' => false,
-            'cancelReasonCode' => null
+            'cancelReasonCode' => $this->getCancelReason()
```

Comment:
Wouldn't the reason code get set as the reason code and the description? Should we separate the two?

Reply:
No, if you take a look at Vindicia's PHP library, you can only specify the code itself, not the description. Which makes sense, since every code can only have 1 description